TRIAGING.md

```diff
@@ -20,7 +20,7 @@ If you have an issue you'd like to bring forth please consider getting a link to
 
 ### Is there an agenda for each week?
 
-Meetings are lightly structured as follows: 
+Meetings are structured as follows: 
```

Comment:
This is good catch. But the general idea of the PR is to keep the changes in the PR to the expected scope. This could be a good separate PR along-with any other changes that you find in the document.

In this particular case we should remove this change from here as 2.x line doesn't contain TRIAGING.md and so the auto-cut backport might fail.

Reply:
Yeah, it was just a change I needed to push because the workflow did not run for some reason the first push.

Reply:
Removed.